spec/factories/case_contacts.rb

```diff
@@ -13,7 +13,7 @@
     association :creator, factory: :user
     casa_case
 
-    contact_types { [create(:contact_type)] }
+    contact_types { [association(:contact_type)] }
```

Comment:
It will play a little nicer with build/build_stubbed [according to the docs](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#has_many-associations), so fewer db hits in those situations.